lib/haml_lint/ruby_extractor.rb

```diff
@@ -187,6 +187,11 @@ def add_line(code, node_or_line, discard_blanks = true)
       end
     end
 
+    def indentation_code(code, indent)
```

Comment:
Fixed it!
https://github.com/brigade/haml-lint/pull/258/commits/164cf9c8a12682e9b972c193becc1bee3a49bfbc